app/views/events/index.html.erb

```diff
@@ -0,0 +1,64 @@
+<p id="notice"><%= notice %></p>
+
+<h1>Event Categories</h1>
+
+<table>
+    <h3>UPCOMING EVENTS</h3>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Description</th>
+      <th>Location</th>
+      <th>Date</th>
+      <th>Creator</th>
+      <th colspan="4"></th>
+    </tr>
+  </thead>
+
+    <% Event.upcoming_events.each do |event| %>
+          <tr>
+            <td><%= event.name %></td>
+            <td><%= event.description %></td>
+            <td><%= event.location %></td>
+            <td><%= event.date %></td>
+            <td><%= event.creator.name %></td>
+            <td><%= link_to 'Show Event', event %></td>
+            <td><%= link_to 'Update Event', edit_event_path(event) %></td>
+            <td><%= link_to 'Cancel Event', event, method: :delete, data: { confirm: 'Are you sure?' } %></td>
+            <td><%= link_to 'Attend Event', event_attending_path(event), method: :post %></td>
+          </tr>
+    <% end %>
+  </tbody>
+
+</table>
+
+<table>
+    <h3>PAST EVENTS</h3>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Description</th>
+      <th>Location</th>
+      <th>Date</th>
+      <th>Creator</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+    <% Event.past_events.each do |event| %>
+          <tr>
+            <td><%= event.name %></td>
+            <td><%= event.description %></td>
+            <td><%= event.location %></td>
+            <td><%= event.date %></td>
+            <td><%= event.creator.name %></td>
+            <td><%= link_to 'Show Event', event %></td>
+            <td><%= link_to 'Update Event', edit_event_path(event) %></td>
+            <td><%= link_to 'Delete Event', event, method: :delete, data: { confirm: 'Are you sure?' } %></td>
+          </tr>
+    <% end %>
+  </tbody>
+
+</table>
```

Comment:
### You can create partial files for the past and upcoming events, with that it will be easier to maintain and read